gravitee-apim-gateway/gravitee-apim-gateway-reactor/src/main/java/io/gravitee/gateway/reactor/spring/ReactorConfiguration.java

```diff
@@ -127,6 +134,22 @@ public PlatformProcessorChainFactory globalProcessorChainFactory(
         );
     }
 
+    @Bean("httpRequestTimeout")
+    public Long httpRequestTimeout(
+        @Value("${http.requestTimeout:30000}") long httpRequestTimeout,
+        @Value("${" + JUPITER_MODE_ENABLED_KEY + ":" + JUPITER_MODE_ENABLED_BY_DEFAULT + "}") boolean isJupiterEnabled
+    ) {
+        if (httpRequestTimeout <= 0) {
+            if (isJupiterEnabled) {
+                log.warn("Http request timeout cannot be set to 0. Setting it to default value: 30_000 ms");
+                httpRequestTimeout = 30_000L;
+            } else {
+                log.warn("A proper timeout should be set in order to avoid unclose connexion, suggested value is 30_000 ms");
+            }
+        }
+        return httpRequestTimeout;
+    }
```

Comment:
I'm wondering if it could be locate this in the `GatewayConfiguration` instead ? But maybe it more a global question  cause it could be an opportunity to centralize a proper configuration (especially to deal with default values that can be hard to maintain with `@Value`).

Reply:
As discussed, I will create a dedicated ticket to think about this case

Reply:
here is the issue: https://github.com/gravitee-io/issues/issues/8206